README.md

```diff
@@ -22,13 +19,14 @@ Supported distributions:
 - CentOS
 - Arch Linux
 - Oracle Linux
+- Opensuse Tumbleweed & Leap
 
 ## Usage
 
 Download and execute the script. Answer the questions asked by the script and it will take care of the rest.
 
 ```bash
-curl -O https://raw.githubusercontent.com/angristan/wireguard-install/master/wireguard-install.sh
+curl -O https://raw.githubusercontent.com/landaal-ict/wireguard-install/master/wireguard-install.sh
```

Comment:
This is definitely not something that should be done.

Reply:
Of course.. but the rest of the code.

Reply:
You should fix your pull request to include only the changes that you want to implement, instead of trying to hijack the readme.